src/reactviews/pages/QueryResult/textView.tsx

```diff
@@ -0,0 +1,220 @@
+                                            valueLength,
+                                        );
+                                    });
+                                }
+
+                                // Apply minimum width of 10 to each column
+                                for (let i = 0; i < columnWidths.length; i++) {
+                                    columnWidths[i] = Math.max(columnWidths[i], 10);
+                                }
+
+                                // Format all data rows with proper column widths
+                                for (const row of response.rows) {
+                                    const formattedRow = row
+                                        .map((cell, index) => {
+                                            const displayValue = cell.isNull
+                                                ? "NULL"
+                                                : cell.displayValue || "";
+                                            return displayValue
+                                                .toString()
+                                                .padEnd(columnWidths[index]);
+                                        })
+                                        .join("  ");
+                                    formattedRows.push(formattedRow);
+                                }
+                            }
+                        } else {
+                            // Apply minimum width of 10 to each column even with no data
+                            for (let i = 0; i < columnWidths.length; i++) {
+                                columnWidths[i] = Math.max(columnWidths[i], 10);
+                            }
+                        }
+
+                        // Add column headers with proper alignment
+                        const headerLine = columnNames
+                            .map((name, index) => name.padEnd(columnWidths[index]))
+                            .join("  ");
+                        content += headerLine + "\n";
+
+                        // Add separator line
+                        const separatorLine = columnWidths
+                            .map((width) => "-".repeat(width))
+                            .join("  ");
+                        content += separatorLine + "\n";
+
+                        // Add the formatted data rows
+                        for (const formattedRow of formattedRows) {
+                            content += formattedRow + "\n";
+                        }
+
+                        // Add row count information
+                        if (resultSetSummary.rowCount > 0) {
+                            content += `(${resultSetSummary.rowCount} row${resultSetSummary.rowCount === 1 ? "" : "s"} affected)\n`;
+                        } else {
+                            content += "(0 rows affected)\n";
+                        }
+
+                        content += "\n";
```

Comment:
`os.EOL`

Reply:
Os is not present in browser environment.